packages/onft-evm/turbo.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": ["//"],
+  "pipeline": {
+    "build": {
+      "outputs": [
+        "dist/**",
+        "artifacts*/**",
+        "hh-cache*/**",
+        "cache*/**",
+        "src/typechain-types/**",
+        "out/**",
+        "node_modules/**"
```

Comment:
I don't think we need node_modules in outputs

Reply:
Shameless copied this file from elsewhere.  Sure, I will remove.